### PR TITLE
docs/user/status: Document RetrievedUpdates reasons

### PR DESCRIPTION
**Pull request by wking**
The messages included in the condition are intended to be fairly clear, but add some additional docs here where we can talk about the impact of the failure and suggest possible mitigations.

### CODE REVIEW

File: docs/user/status.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+### InvalidURI
+
+The configured `upstream` URI is not valid.
+
+Fix by setting `spec.upstream` in ClusterVersion to point to a valid [Cincinnati][] URI, for example https://api.openshift.com/api/upgrades_info/v1/graph .
+
+### InvalidID
+
+The configured `clusterID` is not a valid URI.
```
Comment (vrutkovs) [inline docs/user/status.md]:
```suggestion
The configured `clusterID` is not a valid UUID.
```

Comment (wking) [reply docs/user/status.md]: Thanks.  Fixed in 54b01d4 -> c79a2e4.

**Comment by vrutkovs** [comment]
LGTM
/cc @jottofar

File: docs/user/status.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+The configured `clusterID` is not a valid UUID.
+
+Fix by setting `spec.clusterID` to a valid UUID.
+The UUID should be unique to a given cluster, because it is the default value used for reporting Telemetry and Insights.
+It may also be used by the CVO when making Cincinnati requests, so that Cincinnati can return update recommentations tailored to the specific cluster.
+
+### NoArchitecture
+
+The set of architectures has not been configured.
```
Comment (jottofar) [inline docs/user/status.md]: Why doesn't this "reason" have an explanation of how to fix? This is the only one that does not so it should have something here.

Comment (wking) [reply docs/user/status.md]: > Why doesn't this "reason" have an explanation of how to fix?

I forgot ;).  Added the usual "CVO bug, need an update" boilerplate with c79a2e4 -> 69c81b9.

**Pull request merged**